lib/main.dart

```diff
@@ -143,7 +147,54 @@ class _MyHomePageState extends State<MyHomePage> {
                 ? LoadingInfo(loading)
                 : IconButton(
                     icon: Icon(Icons.menu),
-                    onPressed: () => Scaffold.of(context).openDrawer(),
+                    //onPressed: () => Scaffold.of(context).openDrawer(),
+                    onPressed: () => showModalBottomSheet(
+                      context: context,
+                      shape: RoundedRectangleBorder(
+                        borderRadius: BorderRadius.only(
+                          topLeft: Radius.circular(12),
+                          topRight: Radius.circular(12),
+                        ),
+                      ),
+                      builder: (context) => Container(
+                        padding: EdgeInsets.only(top: 8),
+                        child: Column(
+                          mainAxisSize: MainAxisSize.min,
+                          children: <Widget>[
+                            Padding(
+                              padding: const EdgeInsets.all(8.0),
+                              child: ModalDrawerHandle(),
+                            ),
+                            ListTile(
+                              leading: Icon(Icons.favorite),
+                              title: Text(
+                                'Favorites',
+                                style: TextStyle(
+                                  fontSize: 20,
+                                ),
+                              ),
+                              onTap: () {
+                                _pageNavigatorKey.currentState.pushReplacementNamed('/favorites');
```

Comment:
Here we're getting a bit too much to the right. Might make sense to split into separate widgets, or even just methods.

Reply:
Sheet content extracted to it's own widget in `lib/widgets`